ci/build/builder.go

```diff
@@ -161,6 +167,26 @@ func (build *Builder) Engine(ctx context.Context) (*dagger.Container, error) {
 				"gpg", "curl",
 			}).
 			WithoutEnvVariable("DAGGER_APT_CACHE_BUSTER")
+	case "wolfi":
+		base = dag.
+			Container(dagger.ContainerOpts{Platform: build.platform}).
+			From(consts.WolfiImage).
+			// NOTE: wrapping the apk installs with this time based env ensures that the cache is invalidated
+			// once-per day. This is a very unfortunate workaround for the poor caching "apk add" as an exec
+			// gives us.
+			// Fortunately, better approaches are on the horizon w/ Zenith, for which there are already apk
+			// modules that fix this problem and always result in the latest apk packages for the given alpine
+			// version being used (with optimal caching).
```

Comment:
We can use other modules as dependencies here.

Reply:
Yeah, I meant to say this becomes a lot cleaner if we can use the apk module. I think you wouldn't even need the wolfi-base image, just use scratch and install the wolfi-baselayout package.

Reply:
Yes, what I'm saying is why not add the apk module as a dependency to this ci module and use it here?

Reply:
I don't *think* so? But also, I think it's fairly out-of-scope for this PR.